Landing Page - Sherry

Sherryzh0830 opened the pull request:
## **Contribution:**
A testing page that gives a rough landing for work in progress.

### **Issue:** 

Closes # 598

### **Accomplishments:**
Was able to explore quite a few styling options and tried to make consistent with the existing files. 

## **Details:**

### **Expected Behavior:**
- Have a message to let users know F!rosh is approaching.
- Display logo of F!rosh and able to navigate user to the social media page

### **Complications:**
Renamed and changed font, colors to be consistent to App.scss and unique to the tasks.

Sherryzh0830 committed:
Landing Page - Sherry

natapokie commented on client/src/pages/Initial/LandingPage<Sherry>/LandingPage<Sherry>.scss:
```diff
@@ -0,0 +1,37 @@
+* {
```
when you're working on any styles, make sure to use very unique classnames, for example `landing-page-sz-top`.
in react, all the classnames and their style are shared with all components, so by making very unique classnames you'll be able to ensure that the style you have is not going to affect any components.

additionally, you can take a look at `App.scss`, this is where we define the style for "global" components like `*`, `:root`, `body`. you'll need to rename these in your component and use `className` to define these styles instead.

natapokie commented on client/src/pages/Initial/LandingPage<Sherry>/LandingPage<Sherry>.scss:
```diff
@@ -0,0 +1,37 @@
+  place-content: center;
+  background-color: var(--background-color);
+  min-height: 100vh;
+  font-family: 'Oswald', sans-serif;
```
font-family is also already defined, we're going to stick with the current fonts that we have set up which is `Inter` you can find it in `App.scss` to ensure that we stick with branding guidelines.

since it's already defined "globally" there is no need to add font classes

natapokie reviewed (changes requested):
thanks for sharing your progress so far Sherry! I left some comments that you can take a look at as you complete the issue

also just a quick note, please make sure to rename your files so that they don't contain "<>" and any other special characters! these characters aren't very git friendly haha

the details in the issue weren't too clear, but the angle brackets, `<YourName>` were meant to indicate that you would fill in to replace the angle brackets like this... `LandingPageSherry`.
sorry for the confusion!

Sherryzh0830 committed:
changes to LandingPageSherry

Sherryzh0830 committed:
Edits to SZLandingPage 5/30

Sherryzh0830 committed:
Delete orientation-website.code-workspace

Sherryzh0830 committed:
Delete pull_request_template.md

uzFer reviewed (commented):
My screen looks like this, are there any instructions I'm missing when running your landing page (I just imported and used it in App.jsx)?

![image](https://github.com/UofT-Frosh-Orientation/orientation-website/assets/109243682/44bacaa7-17b8-4d64-86fd-12380625de19)

Sherryzh0830 committed:
Update LandingPageSherry.jsx

Add new instagram link

Sherryzh0830 committed:
Update LandingPageSherry.scss

Styling updates and consistency with colors

Sherryzh0830 committed:
Updated Landing Page SZ

Due to complications, I have modified the landing page and redesigned the styling.

Sherryzh0830 committed:
Updated LandingPageSherry.scss

Resources: https://www.html-code-generator.com/css/speech-bubble-generator
https://www.html-code-generator.com/css/text-shadow-generator

uzFer reviewed (approved):
Works perfectly with the links and visuals, looks really nice!!

natapokie committed:
reverting changes to pull req template

natapokie reviewed (approved):
great work sherry!

for the github file, you actually deleted it, but no worries! i made another commit to this branch to revert the changes, in the future, if you want to revert a file back to a previous commit you can use the following...

```
git checkout [commit ID] -- path/to/file
```
(no square brackets though!)
and then commit as usual!

more git reverting stuff [here](https://dev.to/lofiandcode/git-and-github-how-to-revert-a-single-file-dha#:~:text=look%20like%20this%3A-,git%20checkout%20%5Bcommit%20ID%5D%20%2D%2D%20path/to/file,-If%20I%20were)
